arctic/store/version_store.py

```diff
@@ -837,33 +837,35 @@ def _find_base_version_ids(self, symbol, version_ids):
                                       '_id': {'$nin': version_ids},
                                       'base_version_id': {'$exists': True},
                                       },
-                                     projection=['base_version_id'],
+                                     projection={'base_version_id': 1},
                                      )
         return [version["base_version_id"] for version in cursor]
 
-    def _prune_previous_versions(self, symbol, keep_mins=120, keep_version=None):
+    def _prune_previous_versions(self, symbol, keep_mins=0.01, keep_version=None):
```

Comment:
Why are you turning keep_mins down? 

This is / was needed to handle replication lags etc. Need to ensure this is larger than the maximum lag, or secondary reads can fail.

Reply:
ignore this, I have set it for testing the pruning, when the PR reaches its final form, I will revert

Reply:
I will resolve this conversation once I switch it back.